test/forms/ConvertedPackagingCreditFormProviderSpec.scala

```diff
@@ -68,5 +69,12 @@ class ConvertedPackagingCreditFormProviderSpec extends DecimalFieldBehaviours {
                                    FormError(fieldName, "convertedPackagingCredit.error.required")
       )
     }
+
+    "must reject greater than the allowed amount" in {
```

Comment:
this is probably a duplicate test of the scaffold ones, however as this is specific behaviour i think it makes it explicit to have this test